package.json

```diff
@@ -33,11 +33,6 @@
   "peerDependencies": {
     "vitest": "^3.1.2"
   },
-  "peerDependenciesMeta": {
-    "vitest": {
-      "optional": true
-    }
-  },
```

Comment:
I have checked that all projects that depend on this library are already using vitest as well.